### ISSUE TITLE
[timebox: 12h] Rehearse our contingency plan

### ISSUE THREAD

**Issue by dlapiduz** [18F/cg-product#55]
In order to show how quickly we can recover the platform from scratch, we should measure how long it takes to redeploy into a fresh AWS GovCloud account.

RPO: Platform up, but all S3 and RDS and managed service content is lost

**Acceptance Criteria**

* [ ] Demonstrate whether it's possible to bring up the entire platform in a empty AWS GovCloud account by following the instructions in `cg-provision` in under 12h (aka the Recovery Time Objective, RTO)
* [x] We know how long it takes an undistracted, unfamiliar pair of people to bring it all up
* [ ] The resulting platform is running correctly (eg no b0rked licensing)
* [ ] We have a backlog of "missing pieces" that captures what we'd need in order to bring it up **with** data from a previous deployment

**Implementation Sketch**

**Comment by s-matyukevich** [comment]
https://github.com/18F/cg-bootstrap-concourse-ami/pull/5 tiny pull request, that fixes readme file for cg-bootstrap-concourse-ami repo

**Comment by s-matyukevich** [comment]
I've created a pull request, that contains changes, that allows cg-provision repo to be deployed in non gov regions.
https://github.com/18F/cg-provision/pull/47

For now this pull request is not ready to be merged. I created it to allow everybody to see progress and comment on changes.
The summary of the things that were done:
1. All bucket names made configurable.
2. It becomes possible to configure nat_gateway_ami
3. Tear down jobs uncommitted and parameters fixed for hem  (probably this need to be rolled back, but for now I need this functionality)
4. Links to cg-provision repo changed to my fork (this need to be rolled back before merging)
5. version of 'robdimsdale/concourse-pipeline-resource' update to support new concourse versions.
6. Hardcoded values for bucket names changed in arns.
7. Concourse bootstrap ami_id is made configurable

I also add provision.sh script, that I use to deploy this repository. . For now it is just a log of cli commands, that i use to deploy, but probably later it can be used as a starting point for full deployment automation, This script can be found here https://github.com/s-matyukevich/cg-provision/blob/master/provision.sh

**Comment by mogul** [comment]
Adam says we're waiting on some documentation here.

**Comment by s-matyukevich** [comment]
I was able to deploy cloud gov stack to my corporate AWS account (only staging stack was deployed, because we have AWS limits.)
For now I can't precisely answer on the questions in the check list. I need clean AWS account without restrictions and access to current cloud gov installation in order to do this.
As a result of my job I created a doc, where I collect all issues, that slows down my work. https://docs.google.com/document/d/1HQNTQeTV792Gb8xwheSWVLXEFSjm_X6K2gGKwNDSm5I/edit 
I suggest to review this doc, and probably some of that issues can be added to our board.
